src/app/feed/feed-header/feed-header.component.scss

```diff
@@ -21,7 +21,7 @@
 			font-family: "Courier New";
 			padding:15px 15px;
 			font-size:35px;
-			color: #C8254D;
+			color: #e91e63;
```

Comment:
In accordance with the indigopink theme from angular material. We can add custom theme later. Its just a different shade of pink.